src/lib/process/api.ts

```diff
@@ -0,0 +1,97 @@
+import { Restrictions } from '../types/restrictions.js'
+import { Options } from '../types/options.js'
+import { Polkadot } from '../types/substrate.js'
+import { Process } from './index.js'
+
+export const createProcessTransaction = async (
```

Comment:
Another FYI:

In case fn definition gets bigger it could be created as type

```ts
type CreateProcessTransaction = (polkadot: Polkadot) => Promise<Process>


export const createProcessTransaction: CreateProcessTransaction = async(polkadot) => {
  return Promise.resolve(Process)
}
```